platform/near/transaction.go

```diff
@@ -13,6 +15,14 @@ func (p *Platform) GetTxsByAddress(address string) (types.TxPage, error) {
 	return normalized, nil
 }
 
+func (p *Platform) GetTransactionsByAccount(account, token string, limit int, database *db.Instance) (page types.TxPage, err error) {
```

Comment:
can we get rid of `types.TxPage` type? it's confusing. We should easier use `types.Tx{}` or `types.Txs`

Reply:
Ok, will remove it from golibs